### PR TITLE
Hod/rtir attachment parsing

### PR DESCRIPTION
**Pull request by hod-alpert**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/25329

## Description
Fixed attachment response parsing:
Before it used to return the whole attachment response instead of returning the attachment's content only.

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/RTIR/Integrations/RTIR/RTIR.py [hunk 1]
```diff
@@ -526,43 +526,82 @@ def edit_ticket():
...
+
+    Example output:
+        [('504', 'mimecast-get-remediation-incident.log', 'text/plain', '3.5k'),
+         ('505', 'mimecast-get-remediation-incident2.log', 'text/plain', '3.6k')]
+    Args:
+        raw_attachments: The raw attachments response
+    Returns:
+        A list of tuples containing the id, name, format and size of each attachment
+    """
+    attachments_regex = re.compile(r'(\d+): (.*) \((.*) \/ (.*)\)')
```
Comment (Itay4) [inline Packs/RTIR/Integrations/RTIR/RTIR.py:567]: consider replace `*` with `+`

File: Packs/RTIR/Integrations/RTIR/RTIR_test.py [hunk 2]
```diff
@@ -90,3 +90,41 @@ def test_parse_ticket_links():
...
+
+RAW_ATTACHMENTS_LIST = """
+RT/4.4.2 200 Ok
+
+id: ticket/6325/attachments
+Attachments: 504: mimecast-get-remediation-incident.log (text/plain / 3.5k)
+505: mimecast-get-remediation-incident2.log (text/plain / 3.6k)"""
+
+
+def test_parse_attachments_list():
```
Comment (Itay4) [inline Packs/RTIR/Integrations/RTIR/RTIR_test.py:103]: add docstring to both tests

**Comment by Itay4** [comment]
@hod-alpert please resolve conflicts here

**Comment by kirbles19** [comment]
@hod-alpert Done.

**Pull request merged**